src/js/components/visualization/ChartComponent.jsx

```diff
@@ -114,7 +114,8 @@ export class ChartComponent extends Component {
 
           this._chart.drilldown({
             ...chartData,
-            data: (seriesItem || dataItem)[drillDownFieldName]
```

Comment:
Yeah, they were having that particular case where seriesItem doesn't have the drilldown data, but the dataItem (parent) has it.